src/tasks/tasks.py

```diff
@@ -232,6 +232,12 @@ def n_train_examples(self):
     def n_val_examples(self):
         return self.example_counts["val"]
 
+    @property
+    def no_train_data(self):
+        return self.get_split_text("train") == self.get_split_text("val") or self.get_split_text(
+            "train"
+        ) == self.get_split_text("test")
```

Comment:
This is too dangerous: It's a common and easy sanity check to do an experiment where they load the same file for training and validation and see if the model can get to 100% on that data. This would break that setup. 

Find another way to do this—maybe set it to False in the general task class, and override it explicitly when you need it.

Reply:
Yes, I agree.